src/components/molecules/UiStepper/_internal/UiStepperStep.vue

```diff
@@ -81,27 +82,25 @@ const listItemAttrs = computed(() => ({
     },
   ],
 }));
-const itemTag = computed(() => (isVisitedStep.value
-  ? UiButton
-  : UiText));
-const itemClass = computed(() => (isVisitedStep.value
-  ? [
+const itemClass = computed(() => (
+  [
     'ui-button--text',
     'ui-button--theme-secondary',
+    isInactiveStep.value ? 'ui-button--is-disabled' : '',
```

Comment:
```suggestion
    isInactiveStep.value && 'ui-button--is-disabled',
```
shorter IMO

Reply:
Shorter but less readable.
@mateuszkulpa wdyt?

Reply:
@kubajmarek I prefer the ternary syntax :D 
Also, I don't see any occurrence of `condition && class` in this repo, so it would be inconsistent.